test/github-issues/3534/issue-3534.ts

```diff
@@ -0,0 +1,32 @@
+import 'reflect-metadata';
+import {expect} from 'chai';
+import { Connection, PromiseUtils } from '../../../src';
+import { Foo } from './entity/Foo';
+import { closeTestingConnections, createTestingConnections } from '../../utils/test-utils';
+
+describe('github issues > #3534: store regexp', () => {
+    let connections: Connection[];
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [__dirname + '/entity/*{.js,.ts}'],
+            enabledDrivers: ['postgres'],
+            dropSchema: true,
+        });
+    });
+    after(() => closeTestingConnections(connections));
+
+    it('allows entities with regexp columns', () => PromiseUtils.runInSequence(connections, async connection => {
+        await connection.synchronize();
```

Comment:
we don't usually do it this way... can you please remove it? (just take a look on other tests how we do it in most of the times)...

Reply:
Im just trying to find why tests are passed but CI failed. Usually it happens on a "stucked promise", but I can't easily find the problem in this test